components/Player/Player.jsx

```diff
@@ -83,21 +106,25 @@ const mapDispatchToProps = (dispatch) => ({
   getPlayerMatches: (playerId, numMatches) => dispatch(getPlayerMatches(playerId, numMatches)),
   getPlayerHeroes: (playerId) => dispatch(getPlayerHeroes(playerId)),
   getPlayerWinLoss: (playerId) => dispatch(getPlayerWinLoss(playerId)),
+  getPlayerPeers: (playerId) => dispatch(getPlayerPeers(playerId)),
 });
 
+const getData = props => {
+  props.getPlayer(props.playerId);
+  props.getPlayerMatches(props.playerId, 20);
+  props.getPlayerHeroes(props.playerId);
+  props.getPlayerWinLoss(props.playerId);
+  props.getPlayerPeers(props.playerId);
+};
+
 class RequestLayer extends React.Component {
   componentDidMount() {
-    this.props.getPlayer(this.props.playerId);
-    this.props.getPlayerMatches(this.props.playerId, 20);
-    this.props.getPlayerHeroes(this.props.playerId);
-    this.props.getPlayerWinLoss(this.props.playerId);
+    getData(this.props);
```

Comment:
yea that's what the pages is for, I am going to put all the pages we don't want to preload into there as their own components. I'll push that change up today.